packages/react-menu/Spec.md

```diff
@@ -983,11 +983,37 @@ As a general rule, once a submenu is dismissed without dismissing the menu, the
 
 All of the above Mouse events in the [previous section](#submenu-trigger/navigation) should apply to the part of the split button that is intended to open a submenu.
 
-```
-Keyboard interaction for the split button menu item WIP and requires input from a11y champs
-```
+Below lists the interactactions for the primary content part of the split button
+
+| Type     | Action     | Result  | Details                                          | Focus after                                        |
+| -------- | ---------- | ------- | ------------------------------------------------ | -------------------------------------------------- |
+| Mouse    | Click      | Dismiss | Invoke the menu item                             | User defined - default menu trigger                |
+| Keyboard | Enter      | Dismiss | Invoke the menu item                             | User defined - default menu trigger                |
+| Keyboard | Space      | Dismiss | Invoke the menu item                             | User defined - default menu trigger                |
+| Keyboard | ArrowRight | Open    | Opens the submenu                                | First menu item that contained submenu             |
```

Comment:
small typo in "Focus after" column, I think it would be: "First menu item contained in that submenu"

Reply:
```suggestion
| Keyboard | ArrowRight | Open    | Opens the submenu                                | First menu item contained in that submenu             |
```